### PR TITLE
Spelling: Log in, out, marked noun, small versals

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

**Comment by wopian** [comment]
"Log in as MyAnimeList/Kitsu" does not make sense. The current wording of "Login for MyAnimeList/Kitsu" feels more appropriate.

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -420,19 +420,19 @@
...
     <string name="download_unread">Unread</string>
-    <string name="confirm_delete_chapters">Are you sure you want to delete selected chapters?</string>
+    <string name="confirm_delete_chapters">Are you sure you want to delete the selected chapters?</string>
     <string name="invalid_download_dir">Invalid download location</string>
 
     <!-- Tracking Screen -->
     <string name="manga_tracking_tab">Tracking</string>
     <string name="reading">Reading</string>
-    <string name="currently_reading">Currently Reading</string>
+    <string name="currently_reading">Currently reading</string>
```
Comment (arkon) [inline app/src/main/res/values/strings.xml]: Don't change this, it's meant to match Kitsu.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -420,19 +420,19 @@
...
     <string name="completed">Completed</string>
     <string name="dropped">Dropped</string>
     <string name="on_hold">On hold</string>
     <string name="paused">Paused</string>
     <string name="plan_to_read">Plan to read</string>
-    <string name="want_to_read">Want to Read</string>
+    <string name="want_to_read">Want to read</string>
```
Comment (arkon) [inline app/src/main/res/values/strings.xml]: Don't change this, it's meant to match Kitsu.

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -329,14 +329,14 @@
 
 
     <!-- Login dialog -->
-    <string name="login_title">Login for %1$s</string>
+    <string name="login_title">Log in as %1$s</string>
```
Comment (arkon) [inline app/src/main/res/values/strings.xml]: "Log in to ___"?

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (comradekingu) [reply app/src/main/res/values/strings.xml]: I take it the other one should be "Plan to Read" then?

**Comment by comradekingu** [comment]
@arkon Hoping it looks OK now.
@wopian Added your note as a comment for translators.

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (arkon) [reply app/src/main/res/values/strings.xml]: It's actually shared between different trackers. You know what, it's super inconsistent anyway. Let's just change it...

**Comment by hXtreme** [comment]
Wouldn't "Log in to ..." sound better?

**Pull request merged**